admin/class-admin-init.php

```diff
@@ -61,6 +62,49 @@ public function __construct() {
 	}
 
 	/**
+	 * Validates the extensions and show a notice for the invalid extensions.
+	 */
+	public function validate_extensions() {
+
+		if ( filter_input( INPUT_GET, 'page' ) === WPSEO_Admin::PAGE_IDENTIFIER ) {
+			apply_filters( 'yoast-active-extensions', array() );
```

Comment:
I'm confused why this code is here and what it does.
As I read it an empty array is passed through the filter and is not handled after that.
This does not mean that in all future executions the value is stored and used.

It seems to me that this entire conditional can be removed.